src/utils/datetime.js

```diff
@@ -96,6 +96,14 @@ export function formatClocktimeFromMsPer24(milliseconds, format = 'h:mm a') {
 }
 
 /**
+ * formatCurrentDate
```

Comment:
Take a look at the README inside utils/ and try to follow the instructions for generating a fresh copy of utils API docs (based on the JSDoc comments). Then review [the info about working on docs w/GitBook](http://developer.tidepool.io/docs/docs/workflow.html) and easy have a go at publishing your revised docs, or we can chat about it and review before you try.